terra-rstudio-aou/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.1.2 - 2025-03-07
```

Comment:
Should this changelog and version control be for terra-jupyter-aou instead? I am alsways a bit confused about version control since there is a single image for both jupyter and r.

Would you be able to update the conf.json file accordingly?